tools/bin/build_docker_cache

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+if [ -z "$REPO" ]
```

Comment:
Should we put a sane default here?

Reply:
Especially since we hard code the event type

Reply:
Updated